code/core/src/manager/components/sidebar/ContextMenu.tsx

```diff
@@ -115,7 +161,10 @@ const LiveContextMenu: FC<{ context: API_HashEntry } & ComponentProps<typeof Too
     Addon_TypesEnum.experimental_TEST_PROVIDER
   );
   const providerLinks: Link[] = generateTestProviderLinks(registeredTestProviders, context);
-  const groups = Array.isArray(links[0]) ? (links as Link[][]) : [links as Link[]];
+
+  const groups: Link[][] =
+    Array.isArray(links[0]) || links.length === 0 ? (links as Link[][]) : [links as Link[]];
```

Comment:
We struggled with this, right, we should add a code-comment explaining, so we don't hit the same problems in the future.